### ISSUE TITLE
[✨] RouteLoader which executes only on initial load (not on navigation)

### ISSUE THREAD

**Issue by gnemanja** [BuilderIO/qwik#5835]
### Is your feature request related to a problem?

Right now, if we want to execute code only on inital load, I would have to put it inside useTask.
The problem with this approach is that useTask always waits for routeLoaders to finish. This means that I can't do databasse reads in parallel (those in routeLoaders and in useTask)

### Describe the solution you'd like

Either support for routeLoaders to execute only on initial page load, or implementation of a new concept like initialRouteLoader

### Describe alternatives you've considered

useTask

**Comment by wmertens** [comment]
Actually, this is a duplicate of #2984. Please discuss this enhancement there.
